champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java

```diff
@@ -109,8 +111,12 @@ public void doAlligatorsTooth() {
 
             VelocityData velocityData = new VelocityData(player.getLocation().getDirection(), velocityStrength, false, 0, 0.11, 1.0, true);
             UtilVelocity.velocity(player, null, velocityData);
-            player.getWorld().playEffect(player.getLocation(), Effect.STEP_SOUND, Material.LAPIS_BLOCK);
-            player.getWorld().playSound(player.getLocation(), Sound.ENTITY_FISH_SWIM, 0.8F, 1.5F);
+            
+            if (++soundUpdateCounter % 3 == 0)
```

Comment:
soundUpdateCounter is "global", but is being used as if it is local to players.

Reply:
tbh idk what that means i just copied the code from mineplex, ill try my best though!

Reply:
just make a ticket about the effect playing too frequently and someone else will do it

Reply:
Im telling you guiz listen to realsoy like srsly  windblade sound is terrible 
Idc about tooths sounds but actuallt lapis particles are hella annoying - barely can see anything while diving down the pool

Just turn it back to what it was on mineplex whats the deal

Reply:
@Polisvh ong mineplex was so good at making sounds

Reply:
@RealSoy yes I agree and these nubs dont see it. I will miss scepter sound forever </3

Reply:
new scepter is just stupid in general